enterprise/models.py

```diff
@@ -370,9 +391,10 @@ class Meta:
     )
 
     contact_email = models.EmailField(
+        "Customer admin contact email:",
         null=True,
         blank=True,
-        help_text=_("Email to be displayed as public point of contact for enterprise.")
+        help_text = _("Email address presented on learner portal as public point of contact from customer organization.")
```

Comment:
Can you remove the extra spaces around the equals for this instance and the others? Just to remain consistent